api/src/test/java/org/openmrs/module/bedmanagement/BedManagementServiceTest.java

```diff
@@ -375,4 +376,33 @@ public void shouldSoftDeleteBedTagIfUserHasEditTagsPrivileges() {
 		Assert.assertEquals("Not needed more", bedTag.getVoidReason());
 		Assert.assertTrue(bedTag.getVoided());
 	}
+	
+	@Test
+	public void shouldResizeAdmissionLocationBedLayoutIfUserhasPrivileages() {
+		Context.authenticate(superUser, superUserPassword);
+		
+		BedManagementService bedManagementService = Context.getService(BedManagementService.class);
+		Location location = Context.getService(LocationService.class)
+		        .getLocationByUuid("98bc9b32-9d1a-11e2-8137-0800271c1b75");
+		AdmissionLocation admissionLocation = bedManagementService.getAdmissionLocationByLocation(location);
+		bedManagementService.removeBedLayoutOfAdmissionLocationOutSideRowColumn(admissionLocation, 3, 5);
+		List<BedLocationMapping> bedLocationMappings = bedManagementService.getBedLocationMappingByLocation(location);
+		
+		Assert.assertEquals(15, bedLocationMappings.size());
+		Assert.assertEquals(1, bedLocationMappings.get(0).getRow());
+		Assert.assertEquals(1, bedLocationMappings.get(0).getColumn());
+		Assert.assertEquals(3, bedLocationMappings.get(14).getRow());
+		Assert.assertEquals(5, bedLocationMappings.get(14).getColumn());
+	}
+	
+	@Test(expected = IllegalPropertyException.class)
+	public void shouldThorwErrorOn() {
```

Comment:
There is a typo (`..Thorw..`). Anyway this method is poorly named, please rename it into something meaningful.